src/main/java/org/wso2/carbon/module/swiftiso20022/validation/rules/custom/MT103MultilineFieldValidationRule.java

```diff
@@ -0,0 +1,93 @@
+/**
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.module.swiftiso20022.validation.rules.custom;
+
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.wso2.carbon.module.swiftiso20022.constants.ConnectorConstants;
+import org.wso2.carbon.module.swiftiso20022.constants.MT103Constants;
+import org.wso2.carbon.module.swiftiso20022.validation.common.ValidationResult;
+import org.wso2.carbon.module.swiftiso20022.validation.common.ValidationRule;
+import org.wso2.carbon.module.swiftiso20022.validation.common.ValidatorContext;
+
+import java.util.List;
+
+/**
+ * Validate multiline fields.
+ * Multiline fields are accepted as an array of strings.
+ * Check whether the value is String and String length.
+ */
+public class MT103MultilineFieldValidationRule implements ValidationRule {
```

Comment:
Shall we try to merge this class and repetitive field validation class

Reply:
MT103MultilineFiledValidationRule and MT103RepetitiveFieldValidationRule are merged to MT103ArrayFieldValidationRule in commit 96d4dc498094daee16277501378ee52fbb484dd1